server/utils/json-ld.js

```diff
@@ -88,14 +88,7 @@ export function workLd(content) {
 }
 
 export function museumLd(museum) {
-  const openingHoursSpecification = museum.openingHoursSpecification.map(ohs => {
-    return {
-      dayOfWeek: ohs.dayOfWeek,
-      closes: ohs.closes,
-      opens: ohs.opens
-    };
-  });
-  const newMuseum = Object.assign({}, museum, {openingHoursSpecification, logo: imageLd(museum.logo)});
+  const newMuseum = Object.assign({}, museum);
```

Comment:
What are we actually doing here?

Reply:
Not sure I get the question? I left the Object.assign (after having removed the second object), so that newMuseum is still a clone of the museum object, as we remove a property from it before passing it to objToJsonLd()